test/haml/engine_test.rb

```diff
@@ -2000,7 +2000,7 @@ def test_convert_template_def_method
   end
 
   def test_encoding_error # encoding
-    render("foo\nbar\nb\xFEaz".force_encoding("utf-8"))
+    render("foo\nbar\nb\xFEaz".encode("utf-8"))
```

Comment:
same here https://github.com/haml/haml/blob/770962fec4cfe9ad2d9c6cc676b63d92932e7634/test/cases/encoding_test.rb#L73, this test file is also a fork.